test/mitmproxy/addons/test_cut.py

```diff
@@ -111,6 +111,36 @@ def test_cut_save(tmpdir):
         tctx.command(c.save, "@all", "request.method,request.content", f)
         assert qr(f).splitlines() == [b"GET,content", b"GET,content"]
 
+        with mock.patch("mitmproxy.addons.cut.open") as m:
+            m.side_effect = [PermissionError("Permission denied"),
+                             IsADirectoryError("Is a directory"),
+                             FileNotFoundError("No such file or directory")]
+            for i in range(3):
+                tctx.command(c.save, "@all", "request.method", f)
+            assert tctx.master.has_log("Permission denied", level="error")
+            assert tctx.master.has_log("Is a directory", level="error")
+            assert tctx.master.has_log("No such file or directory",
+                                       level="error")
+
```

Comment:
Doesn't the test below replace the stuff above?

Reply:
\0/
Forget to get rid of it. Sorry.